src/DRAMPower/DRAMPower/memspec/MemSpecLPDDR5.cpp

```diff
@@ -66,7 +66,35 @@ MemSpecLPDDR5::MemSpecLPDDR5(nlohmann::json &memspec)
     }
 
 
-    BGroupMode = (numberOfBankGroups > 1);
+    auto BankArchError = [this]() {
+        std::cout << "Invalid bank architecture selected" << std::endl;
+        std::cout << "Selected values:" << std::endl;
+        std::cout << "  - Number of banks: " << numberOfBanks << std::endl;
+        std::cout << "  - Number of bank groups: " << numberOfBankGroups << std::endl;
+        std::cout << "Valid values are 16|0 (16B mode), 16|4 (BG mode) or 8|0 (8B mode)" << std::endl;
```

Comment:
Actually it should be 16|1, 16|4 or 8|1.

Reply:
fixed